src/udev/udevadm-hwdb.c

```diff
@@ -568,6 +576,7 @@ static int adm_hwdb(struct udev *udev, int argc, char *argv[]) {
         static const struct option options[] = {
                 { "update",  no_argument,       NULL, 'u'     },
                 { "usr",     no_argument,       NULL, ARG_USR },
+                { "strict",  no_argument,       NULL, 's'     },
```

Comment:
This also needs to be added to help() above.

Reply:
Done.